Cargo.toml

```diff
@@ -41,6 +41,7 @@ walkdir = "2.5.0"
 [dev-dependencies]
 rstest = "0.24.0"
 pretty_assertions = "1.4.1"
+lazy_static = "1.5.0"
```

Comment:
Isn’t `lazy_static` already in Rust’s stdlib? I think I remember I used it somewhere.

Found it – [`std::sync::LazyLock`](https://github.com/blockfrost/blockfrost-platform/blob/961ae46e9d0b78a8b770d5238d36e9fd5429279e/src/cbor/fallback_decoder.rs#L314-L316) in @ginnun's PR.